adaptive-insurance/src/app/policy-coverage/page.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import PolicyCoverageContainer from "./PolicyCoverageContainer";
+
+type Props = {};
+
+const page = (props: Props) => {
```

Comment:
Component Name should be Upper-case-first case.
```suggestion
const Page = (props: Props) => {
```